Dockerfile.init

```diff
@@ -15,7 +15,10 @@ ENV PATH ${PATH}:/home/.gimme/versions/gostable.linux.arm64/bin:/home/.gimme/ver
 RUN go version
 
 ENV GO111MODULE on
-RUN go env -w GOPROXY=direct
+
+ARG BUILD_TYPE="RELEASE"
+
+RUN if [ "$BUILD_TYPE"="LOCAL"  ];then go env -w GOPROXY=direct;fi
```

Comment:
typically I see space between the = and the string in bash: https://stackoverflow.com/questions/2237080/how-to-compare-strings-in-bash

I am not sure if this is required or not.

Reply:
yes, this is really good point! But the problem so far is we can't pass BUILD_TYPE=LOCAL to here